log/file_handler.ts

```diff
@@ -30,35 +44,126 @@ export interface FileHandlerOptions extends BaseHandlerOptions {
+   * const handler = new FileHandler("INFO", { filename: "./logs.txt" });
+   * handler.setup();
+   * handler.log('Hello, world!'); // Buffers the message, or writes it to the file depending on buffer state
+   * handler.flush(); // Manually flushes the buffer
+   * handler.destroy(); // Closes the file and removes listeners
+   * ```
+   * **/
   [bufSymbol]: Uint8Array;
+  /** Current position for pointer.
+   * @example Usage
+   * ```ts no-assert
+   * import { FileHandler } from "@std/log/file-handler";
+   *
+   * const handler = new FileHandler("INFO", { filename: "./logs.txt" });
+   * handler.setup();
+   * handler.log('Hello, world!'); // Buffers the message, or writes it to the file depending on buffer state
+   * handler.flush(); // Manually flushes the buffer
+   * handler.destroy(); // Closes the file and removes listeners
+   * ```
+   * **/
   [pointerSymbol] = 0;
+  /** Filename associated with the file being logged.
+   * @example Usage
+   * ```ts no-assert
+   * import { FileHandler } from "@std/log/file-handler";
+   *
+   * const handler = new FileHandler("INFO", { filename: "./logs.txt" });
+   * handler.setup();
+   * handler.log('Hello, world!'); // Buffers the message, or writes it to the file depending on buffer state
+   * handler.flush(); // Manually flushes the buffer
+   * handler.destroy(); // Closes the file and removes listeners
+   * ```
+   * **/
   [filenameSymbol]: string;
+  /** Current log mode.
+   * @example Usage
+   * ```ts no-assert
+   * import { FileHandler } from "@std/log/file-handler";
+   *
+   * const handler = new FileHandler("INFO", { filename: "./logs.txt" });
+   * handler.setup();
+   * handler.log('Hello, world!'); // Buffers the message, or writes it to the file depending on buffer state
+   * handler.flush(); // Manually flushes the buffer
+   * handler.destroy(); // Closes the file and removes listeners
+   * ```
+   * **/
   [modeSymbol]: LogMode;
+  /** File open options.
+   * @example Usage
+   * ```ts no-assert
+   * import { FileHandler } from "@std/log/file-handler";
+   *
+   * const handler = new FileHandler("INFO", { filename: "./logs.txt" });
+   * handler.setup();
+   * handler.log('Hello, world!'); // Buffers the message, or writes it to the file depending on buffer state
+   * handler.flush(); // Manually flushes the buffer
+   * handler.destroy(); // Closes the file and removes listeners
+   * ```
+   * **/
   [openOptionsSymbol]: Deno.OpenOptions;
+  /** Text encoder.
+   * @example Usage
+   * ```ts no-assert
+   * import { FileHandler } from "@std/log/file-handler";
+   *
+   * const handler = new FileHandler("INFO", { filename: "./logs.txt" });
+   * handler.setup();
+   * handler.log('Hello, world!'); // Buffers the message, or writes it to the file depending on buffer state
+   * handler.flush(); // Manually flushes the buffer
+   * handler.destroy(); // Closes the file and removes listeners
+   * ```
+   * **/
   [encoderSymbol]: TextEncoder = new TextEncoder();
```

Comment:
Some of these symbols should probably be private properties, but since they are not, linting requires them to be documented still. I'm not sure how to handle this better.

Reply:
Probably that's the issue of linter (and it could be also an issue of doc generator), but I think it's fine for now for this PR